hypha/apply/funds/tests/test_models.py

```diff
@@ -9,6 +9,7 @@
 from django.core.exceptions import ValidationError
 from django.test import TestCase, override_settings
 from django.urls import reverse
+from django.utils.translation import gettext
```

Comment:
`from django.utils.translation import gettext_lazy as _` is used in other places.

Reply:
Done.

Reply:
@frjo I have done but it is failing for linting as `./hypha/apply/funds/tests/test_models.py:417:13: F402 import '_' from line 12 shadowed by loop variable`